### PR TITLE
add tests for CasePillow (and have travis run ES tests)

### PR DESCRIPTION
**Pull request by czue**
wait for tests

### CODE REVIEW

File: testapps/test_pillowtop/decorators.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    """
+    @functools.wraps(fn)
+    def decorated(*args, **kwargs):
+        assert settings.UNIT_TESTING, 'You can only run this function in unit testing mode.'
+        global TESTS_ALLOWED
+        if not TESTS_ALLOWED and not getattr(settings, 'ALLOW_ELASTICSEARCH_TESTS', False):
+            should_proceed = raw_input(
+                'Are you sure you want to run ElasticSearch tests?? '
+                'These may WIPE all the REAL indices on your computer?? (y/n)\n'
+                'NOTE: set ALLOW_ELASTICSEARCH_TESTS=True in your localsettings to disable this warning.\n').lower()
```
Comment (dimagimon) [inline testapps/test_pillowtop/decorators.py]: E501 line too long (116 > 115 characters)

File: testapps/test_pillowtop/decorators.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            if should_proceed != 'y':
+                print 'Tests aborted!'
+                sys.exit()
+            else:
+                TESTS_ALLOWED = True
+        return fn(*args, **kwargs)
+    return decorated
+
```
Comment (dimagimon) [inline testapps/test_pillowtop/decorators.py]: W391 blank line at end of file

**Comment by czue** [comment]
(don't merge)

**Comment by czue** [comment]
moved to https://github.com/dimagi/pillowtop/pull/152

**Pull request closed without merging**